Documentation/markdown/CMakeOptions.md

```diff
@@ -34,34 +34,35 @@ If you want to see information about external library and thirdparty software op
 | DOXYGEN_OPTIONAL_INCLUDE_COMMENTED_ONLY  | ON      | If selected, only documentation for explicitly commented entities <br>will be generated. Otherwise, doxygen will assume that all entities <br>are documented, even if there is no documentation available.                                                                          | [Go to file](https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake)	|
 | DOXYGEN_OPTIONAL_INCLUDE_INTERNAL        | OFF     | If selected, this will include all comments preceded by "\internal". <br>These are comments intended for developers only. <br>So if the generated documentation is intended for external users, <br>this option should not be selected.                                             | [Go to file](https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake)	|
 | DOXYGEN_OPTIONAL_INCLUDE_EARLY_BINDING   | OFF     | If selected (... and already built), this will include the selected <br>IFC schemas in the documentation. Otherwise, the schemas will not be <br>included in the documentation, but this does not influence the <br>selection of IFC schemas (see EARLYBINDING_ tags in CMake GUI). | [Go to file](https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake)	|
-| DOXYGEN_OPTIONAL_AUTO_OPEN_DOCUMENTATION | ON      | If selected, the landing page of the documentation will open automatically. <br>If not selected, you will have to find and open the generated documentation <br>manually (see section below on building the documentation).                                                         | [Go to file](https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake)	|
+| DOXYGEN_OPTIONAL_AUTO_OPEN_DOCUMENTATION | ON      | If selected, the landing page of the documentation will open automatically. <br>If not selected, you will have to find and open the generated documentation <br>manually.                                                         | [Go to file](https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake)	|
 
-[Go back to overview of project options](#project-options)
+[Go back to overview](#project-options)
 
 ### EarlyBinding
+These options allow you to select which IFC schemas should be built for the project. These options will only be displayed once *OpenInfraPlatform.Commands.UpdateBoostMPL* and *OpenInfraPlatform.ExpressBindingGenerator* have been built (see [SetupHelp.md](./SetupHelp.md)). Once the desired schema or schemas are selected, you have to generate CMake again. The default value for the schemata are set [here](https://github.com/tumcms/Open-Infra-Platform/blob/development/ExpressBindingGenerator/CMakeLists.txt).
```

Comment:
The options are shown nevertheless. One does not need to build either project.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/111/commits/83c90493783a8968a8ff002eff0cd8c52883d463